src/cryptography/hazmat/bindings/openssl/engine.py

```diff
@@ -22,6 +22,7 @@
 typedef ... *ENGINE_GEN_INT_FUNC_PTR;
 typedef ... *ENGINE_CTRL_FUNC_PTR;
 typedef ... *ENGINE_LOAD_KEY_PTR;
+typedef ... *ENGINE_SSL_CLIENT_CERT_PTR;
```

Comment:
k

On Sat, May 16, 2015 at 5:24 PM, Paul Kehrer <notifications@github.com>
wrote:

> In src/cryptography/hazmat/bindings/openssl/engine.py
> <https://github.com/pyca/cryptography/pull/1965#discussion_r30464912>:
>
> > @@ -22,6 +22,7 @@
> >  typedef ... *ENGINE_GEN_INT_FUNC_PTR;
> >  typedef ... *ENGINE_CTRL_FUNC_PTR;
> >  typedef ... *ENGINE_LOAD_KEY_PTR;
> > +typedef ... *ENGINE_SSL_CLIENT_CERT_PTR;
>
> We'll need it for that next PR so seems like no big deal to include it.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pyca/cryptography/pull/1965/files#r30464912>.
>



-- 
"I disapprove of what you say, but I will defend to the death your right to
say it." -- Evelyn Beatrice Hall (summarizing Voltaire)
"The people's good is the highest law." -- Cicero
GPG Key fingerprint: 125F 5C67 DFE9 4084